<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsTimerDialog</class>
 <widget class="QDialog" name="SettingsTimerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Timer Settings</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkSound">
     <property name="text">
      <string>Play sound when timer finishes</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkAction">
     <property name="text">
      <string>Run program or open file</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widgetFileSelect" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEditPath"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnBrowse">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkContinueOnExit">
     <property name="text">
      <string>Continue timer after exit</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
